projects/example-app/src/app/app.config.ts

```diff
@@ -1,40 +1,34 @@
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-import { HttpClientModule } from '@angular/common/http';
-import { BrowserModule } from '@angular/platform-browser';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ApplicationConfig } from '@angular/core';
+import { provideHttpClient } from '@angular/common/http';
+import { provideAnimations } from '@angular/platform-browser/animations';
 
-import { StoreModule } from '@ngrx/store';
-import { EffectsModule } from '@ngrx/effects';
-import { StoreRouterConnectingModule } from '@ngrx/router-store';
-import { StoreDevtoolsModule } from '@ngrx/store-devtools';
-
-import { AuthModule } from '@example-app/auth';
+import { provideState, provideStore } from '@ngrx/store';
+import { provideEffects } from '@ngrx/effects';
+import { provideRouterStore } from '@ngrx/router-store';
+import { provideStoreDevtools } from '@ngrx/store-devtools';
 
 import { rootReducers, metaReducers } from '@example-app/reducers';
 
-import { CoreModule } from '@example-app/core';
-import { AppRoutingModule } from '@example-app/app-routing.module';
+import { APP_ROUTES } from '@example-app/app.routing';
 import { UserEffects, RouterEffects } from '@example-app/core/effects';
-import { AppComponent } from '@example-app/core/containers';
+import { provideRouter, withHashLocation } from '@angular/router';
+import * as fromAuth from '@example-app/auth/reducers';
+import { AuthEffects } from './auth/effects';
 
-@NgModule({
-  imports: [
-    CommonModule,
-    BrowserModule,
-    BrowserAnimationsModule,
-    HttpClientModule,
-    AuthModule,
-    AppRoutingModule,
+export const appConfig: ApplicationConfig = {
+  providers: [
+    provideAnimations(),
```

Comment:
This can be `provideAnimationsAsync` for a small perf improvement.

```suggestion
    provideAnimationsAsync(),
```

Reply:
Done